src/hooks/useCitySearchQuery.jsx

```diff
@@ -0,0 +1,30 @@
+import { useCallback, useMemo } from 'react';
+import { useSearchParams } from 'react-router-dom';
+
+const CITY_SEARCH_QUERY = 'query';
```

Comment:
The idea was to have it as constant to reuse it.

The usage across the project should be like this 
**before**
```js
search: `query=${cityName}`
```
after
```
search: createSearchParams({
                [CITY_SEARCH_QUERY]: [countryName],
              }).toString()
```
or at least
```
search: `${CITY_SEARCH_QUERY}=${cityName}`
```